bundler.d/puma.rb

```diff
@@ -0,0 +1,3 @@
+group :puma do
+  gem 'puma', '> 4.0', :require => 'puma'
```

Comment:
Can we limit to to the major version? Also wondering why the `require` is needed
```suggestion
  gem 'puma', '~> 4.0', :require => 'puma'
```

Reply:
Looks like only 4.1.0 can disable TLS v1.1.
```suggestion
  gem 'puma', '~> 4.1', :require => 'puma'
```

Note that in our current packaging we ship 3.11.4 so that needs an update as well.

Reply:
> Note that in our current packaging we ship 3.11.4 so that needs an update as well.

https://github.com/theforeman/foreman-packaging/commit/6cbd652262b884f4e391ddb98538c756f859219c bumped puma to 4.3.3

Reply:
Will do thanks.